src/components/FreeType/FreeType.scss

```diff
@@ -28,7 +28,7 @@
 
     line-height: 160%;
 
-    color: $color-grey-dark; //#667085;
+    color: $color-blue-navy; //#667085;
```

Comment:
Please get rid of the hex code and run npm run format again before pushing it back up